ansible/kepler_playbook.yml

```diff
@@ -1,5 +1,5 @@
 - name: Setup Kepler and OpenMetrics
-  hosts: all
+  hosts: localhost
```

Comment:
wouldn't this affect the other playbooks? isn't there way to select host from command line?